docs/continuous-integration.md

```diff
@@ -270,20 +282,42 @@ you also need to add
 
 ### Bitbucket
 
-#### Step 1: Generate an app password
+To allow Happo to post build statuses to your Bitbucket repo, you need to
+configure Happo with an
+[API token](https://support.atlassian.com/bitbucket-cloud/docs/using-api-tokens/)
+or a
+[Repository access token](https://support.atlassian.com/bitbucket-cloud/docs/repository-access-tokens/).
 
-To authorize Happo to post statuses to your PRs/commits, you need to generate an
-[app password](https://support.atlassian.com/bitbucket-cloud/docs/app-passwords/).
+#### Step 1: Generate a token
+
+##### Alternative A: API Token
 
-![Generating a Bitbucket app password](/img/happo-bitbucket-app-password.gif)
-_Generating an app password through the Bitbucket UI_
+On your
+[API tokens page for your profile](https://id.atlassian.com/manage-profile/security/api-tokens),
+generate
+[a scoped API token](https://support.atlassian.com/bitbucket-cloud/docs/using-api-tokens/)
+with the scope `read:repository:bitbucket`.
+
+![Generating a scoped Bitbucket API token](/img/happo-bitbucket-api-token.gif)
+_Generating a scoped Bitbucket API token through the Bitbucket UI_
+
+##### Alternative B: Repository access token
+
+On your repository settings page, go to `Security > Access tokens`. Generate a
+token with the `Repositories > Read` scope.
+
+![Generating a repository access token](/img/happo-bitbucket-repo-access-token.gif)
```

Comment:
Are we able to use video files instead of gif for these? I think the quality would be much better and the filesize would be much smaller. I'll give it a shot for something and see if I can get a video to work well here.